src/clr-angular/forms/common/providers/control-class.service.ts

```diff
@@ -1,22 +1,25 @@
 /*
- * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import { Injectable } from '@angular/core';
+import { Injectable, Optional } from '@angular/core';
+import { LayoutService } from './layout.service';
 
 @Injectable()
 export class ControlClassService {
   className = '';
 
-  controlClass(invalid = false, grid = false, additional = '') {
+  constructor(@Optional() private layoutService: LayoutService) {}
+
+  controlClass(invalid = false, grid = false, labelSize = 2, additional = '') {
     const controlClasses = [this.className, additional];
     if (invalid) {
       controlClasses.push('clr-error');
     }
     if (grid && this.className.indexOf('clr-col') === -1) {
```

Comment:
We need to check if layoutService is defined in this `if` block before we use it.